server/src/com/openrsc/server/io/WorldLoader.java

```diff
@@ -44,18 +46,29 @@ private static boolean projectileClipAllowed(final int wallID) {
 
 	private boolean loadSection(final int sectionX, final int sectionY, final int height, final int bigX, final int bigY) {
 		Sector s = null;
-		try {
-			final String filename = "h" + height + "x" + sectionX + "y" + sectionY;
-			final ZipEntry e = tileArchive.getEntry(filename);
-			if (e == null) {
-				//LOGGER.warn("Ignoring Missing Sector: " + filename);
+
+		if (jagArchive != null) {
```

Comment:
This is fine, but maybe would have been better to be explicit & read some property of JContent for what version of the map the jagArchive contains. 👍

Reply:
We can't afaik, the content files have used the same format since the beggining of time. Obfuscated BZLib is an implementation based on bzip back in the day they used, I know they use BZip2 for a fact in later versions, and we will need to check for that based on config most likely of what the user say version they are using.

Reply:
Yeah, that works, sounds good.